src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -729,12 +726,16 @@ private void RefreshDependencyNames()
 
         private void RefreshPackageContents()
         {
-            PackageContents = CustomNodeDefinitions.Select(
-                    (def) => new PackageItemRootViewModel(def))
```

Comment:
Not sure how this ever worked - `ObservableCollections` cannot be set AFAIK but only update in response to Add, Remove, Replace or Clear.